### ISSUE TITLE
Some commands seem to be missing data

### ISSUE THREAD

**Issue by wezm** [pimoroni/inky#6]
I've been reviewing this code to see how to control the display that I have (Inky pHAT Black/White/Red v2) and noticed what I think are some inconsistencies. Granted that some of my information is based upon my assumption that the display is using an SSD1675 controller, which could be wrong but all the other commands match.

Is gate driving voltage being set correctly? It seems that the two bytes should be ORed together instead of being sent as separate bytes.

<https://github.com/pimoroni/inky/blob/b19641e9c4d6d88cc3a184d065a0d93605bbcfc1/library/inky/inky.py#L164>

![image](https://user-images.githubusercontent.com/21787/48525614-ea858900-e8d8-11e8-96ee-9a33920befd7.png)

----

I think this command is missing the three data bytes. I note that [they are present in the old library](https://github.com/pimoroni/inky-phat/blob/4c0ac9ffae25d2ee055f41f1d958c64b17f574bd/library/inkyphat/inky212x104.py#L306).

<https://github.com/pimoroni/inky/blob/b19641e9c4d6d88cc3a184d065a0d93605bbcfc1/library/inky/inky.py#L170>

![image](https://user-images.githubusercontent.com/21787/48525730-49e39900-e8d9-11e8-93c9-35de0bd365e0.png)

**Comment by Gadgetoid** [comment]
I'll see if we can dig up the datasheet- there's a good chance you're right!

**Comment by chrislast** [comment]
based on https://www.china-epaper.com/uploads/soft/DEPG0420R01V3.0.pdf I'd agree

**Comment by Gadgetoid** [comment]
Having - finally - found some time to look into this, I also concur. I've created a new branch for addressing any tweaks to bring the registers in line with the datasheet. Quite a few things got confused between the old/new versions of Inky pHAT, which used somewhat different controllers.

I've raised a draft PR to track changes here- https://github.com/pimoroni/inky/pull/25

I'll aim to revert values to their defaults and experiment with display updates from there. It may be confirmation bias, but I believe a higher Gate Driving Voltage yields slightly better colour contrast on the Yellow and Red Inky boards.
